Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: Bump quickcheck dependencies to 1

We really don't want to have to bring an old version of quickcheck
back into the Debian archive for this crate.

see also https://github.com/capnproto/capnproto-rust/pull/275

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,4 +37,4 @@
 default-features = false
 
 [dev-dependencies.quickcheck]
-version = "0.9"
+version = "1"
